spec/models/instrument_spec.rb

```diff
@@ -488,6 +488,15 @@
       assert_equal 45, @next_reservation.reserve_start_at.min
     end
 
+    it "should find next available reservation with cutoff hours" do
+      expect(@rule.instrument.update_attribute :cutoff_hours, 10).to be true
+
+      @next_reservation = @instrument.next_available_reservation(after = Time.zone.now.beginning_of_day)
```

Comment:
Do we need the `after` assignment? Does `next_reservation` need to be an instance variable?